wordpress-mobile/WordPress-iOS
# Fix Me screen when logged in with self hosted site

Fixes #12809 

To test:
- Log out if necessary.
- Log in with a self-hosted site.
- Verify the 'Me' view is not blank.
- Test on iPhone and iPad please.

| Before | After |
|--------|-------|
|        |       |
| ![before_iphone](https://user-images.githubusercontent.com/1816888/67907879-18da9380-fb3f-11e9-8ee2-6ab60063d6cb.png)| ![after_iphone](https://user-images.githubusercontent.com/1816888/67907895-26901900-fb3f-11e9-9ce3-9f37e7adc77e.png)|
|![before_ipad](https://user-images.githubusercontent.com/1816888/67907907-3576cb80-fb3f-11e9-941a-d6e0c8dc704e.png)|![after_ipad](https://user-images.githubusercontent.com/1816888/67907919-3e679d00-fb3f-11e9-9d85-5f781213c85e.png)|

Update release notes:
